packages/client/src/components/Accordion/index.js

```diff
@@ -68,8 +82,11 @@ const AccordionWrapper = styled.div`
   background-color: ${COLORS.white};
   height: auto;
   font-size: 14px;
+  font-family: ${FONTS.font2};
   color: black;
   min-width: 220px;
```

Comment:
How did 220px get decided?

Reply:
oh this was based on the figma, but pretty arbitrary